orion/controllers/programs/programsController.go

```diff
@@ -33,13 +34,10 @@ func CreateProgram(c *gin.Context) {
   var programJson Program
   c.BindJSON(&programJson)
 
-  // TODO: implement for real!
-  // isValid := CheckValidProgram(newProgram);
-  // if (isValid) {
-  //
-  // } else {
-  //
-  // }
+  // TODO: test if this works
+  if CheckValidProgram(c) == false {
```

Comment:
Look below about my comments regarding `gin`. You won't need it yet.
Instead, the parameter here should be a `Program`.